systray/systray_real.go

```diff
@@ -96,25 +93,41 @@ func (s *Systray) start() {
 			case <-mRmCrashes.ClickedCh:
 				RemoveCrashes()
 				s.updateMenuItem(mRmCrashes, config.LogsIsEmpty())
-			case <-mGenCerts.ClickedCh:
+			case <-mManageCerts.ClickedCh:
+				infoMsg := "The Arduino Agent needs a local HTTPS certificate to work correctly with Safari.\n\nYour HTTPS certificate status:\n"
+				buttons := "{\"Install the certificate for Safari\", \"OK\"} default button \"OK\""
 				certDir := config.GetCertificatesDir()
-				cert.GenerateCertificates(certDir)
-				err := cert.InstallCertificate(certDir.Join("ca.cert.cer"))
-				// if something goes wrong during the cert install we remove them, so the user is able to retry
-				if err != nil {
-					log.Errorf("cannot install certificates something went wrong: %s", err)
-					cert.DeleteCertificates(certDir)
-				}
-				s.Restart()
-			case <-mRemoveCerts.ClickedCh:
-				err := cert.UninstallCertificates()
-				if err != nil {
-					log.Errorf("cannot uninstall certificates something went wrong: %s", err)
+				if config.CertsExist() {
+					expDate, err := cert.GetExpirationDate()
+					if err != nil {
+						log.Errorf("cannot get certificates expiration date, something went wrong: %s", err)
+					}
+					infoMsg = infoMsg + "- Certificate installed: Yes\n- Certificate trusted: Yes\n- Certificate expiration date: " + expDate
```

Comment:
Would be nice to remove the " +0000" text here too (in addition to the expiration check).
I would move the "replace fix" inside of `GetExpirationDate`, so that it's available in all places.